src/main/java/httpserver/HTTPResponse.java

```diff
@@ -0,0 +1,22 @@
+package httpserver;
+
+public class HTTPResponse {
+
+    private String statusCode;
+
+    public HTTPResponse(int statusCode) {
+        this.statusCode = String.valueOf(statusCode);
+    }
+
+    public String HTTP() {
+        return "HTTP/1.1";
+    }
+
+    public String getStatusCode() {
+        return statusCode;
+    }
+
+    public String getReasonPhrase() {
+        return "OK";
```

Comment:
Yep, since I was only really trying to get a simple get request to return a 200 response I just hard coded it, it will change!